provider/resources.go

```diff
@@ -148,10 +153,30 @@ func Provider() tfbridge.ProviderInfo {
 			Namespaces: map[string]string{
 				mainPkg: "Kafka",
 			},
-		},
+		}, MetadataInfo: tfbridge.NewProviderMetadata(metadata),
+	}
+
+	err := x.ComputeDefaults(&prov, x.TokensSingleModule("kafka_", mainMod,
+		x.MakeStandardToken(mainPkg)))
+	contract.AssertNoErrorf(err, "ComputeDefaults may have failed")
+
+	docLessResources := []string{
+		"kafka_user_scram_credential",
 	}
 
+	missingDocInfo := &tfbridge.DocInfo{
+		Markdown: []byte{' '},
+	}
+	for _, s := range docLessResources {
+		prov.Resources[s].Docs = missingDocInfo
+	}
+
+	err = x.AutoAliasing(&prov, prov.GetMetadata())
+	contract.AssertNoErrorf(err, "auto aliasing apply failed")
+
 	prov.SetAutonaming(255, "-")
 
 	return prov
 }
+
```

Comment:
This needs a doc comment using `go:embed`.
```suggestion
//go:embed cmd/pulumi-resource-kafka/bridge-metadata.json
```